mirgecom/simutil.py

```diff
@@ -507,14 +517,18 @@ def compare_files_vtu(
             raise ValueError("Fidelity test failed: Mismatched data array sizes")
 
         # verify individual values w/in given tolerance
-        print(f"Field: {point_data2.GetArrayName(i)}", end=" ")
+        fieldname = point_data1.GetArrayName(i)
+        print(f"Field: {fieldname}", end=" ")
         for j in range(arr1.GetSize()):
             test_err = abs(arr1.GetValue(j) - arr2.GetValue(j))
             if test_err > max_field_errors[i]:
                 max_field_errors[i] = test_err
-        print(f"Max Error: {max_field_errors[i]}")
+        print(f"Max Error: {max_field_errors[i]}", end=" ")
+        print(f"Tolerance: {configurate(fieldname, field_tol, tolerance)}")
+
+    violation = any([max_field_errors[i] > configurate(fieldname, field_tol,
+                                        tolerance) for i in range(nfields)])
```

Comment:
I think `fieldname` should be `point_data1.GetArrayName(i)` in this violation check.

Reply:
Come to think of it, perhaps something like:
```
field_tols = [configurate(point_data1.GetArrayName(i), field_tol, tolerance) for i in range(nfields)]
(...)
       print(f"Tolerance: {field_tols[i]}")
 violation = any([max_field_errors[i] > field_tols[i] for i in range(nfields)])
```

Reply:
Ooh, that makes a lot of sense. 
I have this right now but it can't compare between max_field_errors[I] and field_tol[I]... fieldname is point_data1.GetArrayName(i) here:
```
   field_tol[i] = [configurate(fieldname, field_tol, tolerance) for i in range(nfields)]
          print(f"Tolerance: {field_tol[i]}")

violation = any([max_field_errors[i] > field_tol[i] for i in range(nfields)])
```
Output:
```
 violation = any([max_field_errors[i] > field_tol[i] for i in range(nfields)])
TypeError: '>' not supported between instances of 'int' and 'list'
```

Reply:
Woops!  Note that above I have a new variable `field_tols` which is a new _list_ variable containing all the field-specific tolerances (or the default if they did not exist).  Then I use `field_tols` in the violation check *not* the old `field_tol` which was (and should remain) a dictionary.

Reply:
Ah my bad I just assumed they were the same list. The second block of code you mentioned throws error that the violation bool is not iterable but the first block of code works!

Reply:
Sorry about that!  Ha, well I'm not great at Python.  I should try things out before suggesting them!  Thanks for trying it out!